etebase_server/urls.py

```diff
@@ -22,4 +21,4 @@ def serve_static(request, path):
 
         return serve(request, basename, dirname)
 
-    urlpatterns += [re_path(r"^static/(?P<path>.*)$", serve_static)]
+    urlpatterns += [path("static/<path>", serve_static)]
```

Comment:
```suggestion
    urlpatterns += [path("static/<path:str>", serve_static)]
```
I think it should look something like this, not 100% sure.

Reply:
I'm actually having a hard time hitting that route, even with the old code - all of `/static/*` seems to get swallowed by something else before hitting this dispatcher, for `GET /static/foo` the log just shows `INFO:      - "GET /foo HTTP/1.1" 404 Not Found`, and the `serve_static` handler isn't called. `debug` is enabled in the config.

Reply:
Try:
```suggestion
    urlpatterns += [path("static/<path:path>", serve_static)]
```
I think this may fix it.

Reply:
Great, it's gone!